### ISSUE TITLE
It's still hard to find this repo

### ISSUE THREAD

**Issue by ifohancroft** [aperezdc/termite#2]
This repo is unfortunately, still hard to find. Perhaps it would be worth it creating an org for it so it can be found under termite/termite?
I presume this would make it easier to find and hopefully GitHub would prefer showing it in search as opposed to thestinger/termite

**Comment by aperezdc** [comment]
@ifohancroft I have no idea how GitHub prefers to classify search results, but I suppose creating an organization is something we could try. I'll give it a bit of thought and get back to this issue later, thanks for the idea!

**Comment by micwoj92** [comment]
You could try to contact github support to "unfork" the repo. This would make it visible in search by default alongside enabling some additional github features.

**Comment by ifohancroft** [comment]
This is interesting. I didn't know forks show up less (or maybe not at all?) in search results?
I still think creating an organization with the same name and a repo in it would be the best way. I guess ideally, if this particular repo can be unforked and moved into the organization.

**Comment by micwoj92** [comment]
Yes, it would be good to good termite/termite, but it's @aperezdc who has all the powers here.
